pygmt/src/plot.py

```diff
@@ -213,6 +214,18 @@ def plot(self, x=None, y=None, data=None, size=None, direction=None, **kwargs):
         and data.geom_type.isin(["Point", "MultiPoint"]).all()
     ):  # checking if the geometry of a geoDataFrame is Point or MultiPoint
         kwargs["S"] = "s0.2c"
+    elif (
+        "S" not in kwargs and kind == "file"
+    ):  # chacking that the data is a file path to set defualt style
+        try:
+            with open(which(data), "r") as file:
```

Comment:
Need to set an encoding for [PEP597](https://www.python.org/dev/peps/pep-0597/).
```suggestion
            with open(which(data), mode="r", encoding="utf8") as file:
```